Hologram/Network/Modem/EC21.py

```diff
@@ -0,0 +1,199 @@
+# EC21.py - Hologram Python SDK Quectel EC21 modem interface
+#
+# Author: Hologram <support@hologram.io>
+#
+# Copyright 2016 - Hologram (Konekt, Inc.)
+#
+#
+# LICENSE: Distributed under the terms of the MIT License
+#
+import binascii
+import time
+
+from serial.serialutil import Timeout
+
+from Hologram.Network.Modem import Modem
+from Hologram.Event import Event
+from UtilClasses import ModemResult
+from Exceptions.HologramError import SerialError, NetworkError
+
+DEFAULT_EC21_TIMEOUT = 200
+
+class EC21(Modem):
+    usb_ids = [('2c7c', '0121')]
+
+    def __init__(self, device_name=None, baud_rate='9600',
+                 chatscript_file=None, event=Event()):
+
+        super().__init__(device_name=device_name, baud_rate=baud_rate,
+                                        chatscript_file=chatscript_file, event=event)
+        self._at_sockets_available = True
+        self.urc_response = ''
+
+    def connect(self, timeout=DEFAULT_EC21_TIMEOUT):
+
+        success = super().connect(timeout)
+
+        # put serial mode on other port
+        # if success is True:
```

Comment:
can we just get rid of the commented out stuff?

Reply:
how many AT ports does the EC21 have? we could uncomment it and switch AT ports if possible

Reply:
I think just one. The other one seems to be a qualcomm debug port